docs/development.md

```diff
@@ -167,6 +167,8 @@ echo "Machine SSH key generated in ${SSH_KEY_FILE}"
 export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
 ```
 
+If this variable is not set it will generate a new ssh key and use the generated one.
```

Comment:
@cpanato this is in the "Manual Testing" part of the docs. Can you please add a short section about E2E (don't think we have one yet) below. Something like:

```### Running E2E locally

To run E2E locally, set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, `AZURE_TENANT_ID` and run:

#```bash
./scripts/ci-e2e.sh
#```

You can optionally set `AZURE_SSH_PUBLIC_KEY_FILE` to use your own ssh key. If this variable is not set the tests will generate a new ssh key and use the generated one.
```
(ignore the #s above)

Reply:
Done :)